src/citation/views/citation_entry_view.py

```diff
@@ -23,6 +26,40 @@ def list(self, request):
 
     def retrieve(self, request):
         pass
+    
+    @action(detail=False, methods=["post"], permission_classes=[IsAuthenticated])
+    def pdf_uploads(self, request):
+        pdfs = request.FILES.getlist('pdfs[]')
+        schema = generate_schema_for_citation(JOURNAL_ARTICLE)
+        entry_json = {}
+        created = []
+        for pdf in pdfs:
+            url = upload_to_s3(pdf, 'citation_pdfs')
+            path = url.split('.com/')[1]
+            handle_creating_citation_entry.apply_async(
```

Comment:
Let's add a `countdown` for celery tasks.

Reply:
This prevents celery from instantly consuming the task (syncing issues). Anything greater than 1 second is probably good enough.